src/utils/data-management/downloadSingleFile.js

```diff
@@ -1,14 +1,9 @@
-import { saveAs } from 'file-saver';
-import { Storage } from 'aws-amplify';
+import downloadFromUrl from 'utils/data-management/downloadFromUrl';
+import { getFromApiExpectOK } from 'utils/getDataExpectOK';
 
-const downloadSingleFile = async (activeProjectUuid, sampleUuid, fileName, bundleName) => {
```

Comment:
We were using amplify? And it was failing because the permissions to access this from amplify had already been removed?